lib/controllers/v1/observations_controller.js

```diff
@@ -768,28 +768,41 @@ ObservationsController.reqToElasticQueryComponents = function( req, callback ) {
 ObservationsController.resultsForRequest = function( req, callback ) {
   ObservationsController.elasticResults( req, function( err, data ) {
     if( err ) { return callback( err ); }
-    var obs = _.map( data.hits.hits, function( h ) {
-      return new Observation( h._source, { userSession: req.userSession } );
-    });
-    if( err ) { return callback( err ); }
-    var response = { total_results: data.hits.total };
-    if (
-      data.aggregations &&
-      data.aggregations.bbox &&
-      data.aggregations.bbox.bounds &&
-      data.aggregations.bbox.bounds.bottom_right
-    ) {
-      response.total_bounds = {
-        swlat: data.aggregations.bbox.bounds.bottom_right.lat,
-        swlng: data.aggregations.bbox.bounds.top_left.lon,
-        nelat: data.aggregations.bbox.bounds.top_left.lat,
-        nelng: data.aggregations.bbox.bounds.bottom_right.lon
-      };
+    function handleObs( obs ) {
```

Comment:
one nitpick of mine is that I don't generally like to define functions within other functions. I'd personally prefer to extract handleObs as a method of ObservationsContoller (one that doesn't need to be exposed via exports)